task_templates/2_estimators/1_python_regression/custom.py

```diff
@@ -13,7 +13,7 @@
 
 
 class CustomTask(RegressionEstimatorInterface):
-    def fit(self, X: pd.DataFrame, y: pd.Series, **kwargs) -> None:
+    def fit(self, X: pd.DataFrame, y: pd.Series, parameters=None, **kwargs) -> None:
```

Comment:
Be sure to update docstring in all changed places to reflect new `parameters` parameter